Xwt/Xwt/Toolkit.cs

```diff
@@ -375,6 +375,10 @@ public object ValidateObject (object obj)
 		{
 			if (obj is Image)
 				((Image)obj).InitForToolkit (this);
+			else if (obj is TextLayout)
+				((TextLayout)obj).InitForToolkit (this);
+			else if (obj is Font)
+				return ((IFrontend)obj).ToolkitEngine != this ? Font.FromName (obj.ToString (), this) : obj;
```

Comment:
Only place that calls ValidateObject for a font right now is `TextLayout` because I'm assuming that's the only place that will in my situation (foreign toolkit drawing).  Because of that, that call would only be used once via the `InitForToolkit` check of `TextLayout`.

Originally I had that code directly in the `TextLayout` class, I can move it back there if you want. Otherwise should we go for the same `InitForToolkit` style of things?